pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -422,6 +426,10 @@ public void run() {
       Configurator.setAllLevels("", logLevel);
     }
 
+    if (metricsSystem == null) {
+      metricsSystem = PrometheusMetricsSystem.instance();
+    }
```

Comment:
When would `metricsSystem` not be null here?

Reply:
It was an early attempt to address my server test breakage.